src/EdozoLogo2/EdozoLogo2.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ProductName } from './EdozoLogo2.styles';
+import { ReactComponent as Logo } from './edozo-logo.svg';
+
+export interface Props {
+  appName?: 'Maps' | 'Insight' | 'Occupiers' | 'Help centre' | '';
```

Comment:
It seems weird to have empty string in here? Can we remove empty string and just have it default to maps maybe? unless there is a reason to put empty string in there